google/earthenterprise
# #566 Update PostgreSQL command interfaces and clean up SQL files

Partial implementation for #566 

Verification Steps:

1. Remove /tmp/fusion_os_install if it exists
1. Make sure at least one fusion database is published
1. Test running install_server.sh while server is still installed
    * It should backup and restore the postgres databases
    * All published fusion databases should still be published
1. Test running install_server.sh without server installed
1. With server installed, publish a database
1. Test running `sudo -u gepguser /opt/google/bin/geresetpgdb` with the following parameters
    1. soft: It should back up the databases and output the location of the backup; it will also replace the databases with initial blank versions
    1. restore <path of backup>: Using the backup path output from the "soft" command, this should restore the databases to their previous state before running the "soft" command, fusion databases should be published
    1. hard: It should completely wipe out the data in the database and start fresh. No databases will be published

Note: I didn't add any code to remove the sql files from /opt/google/bin
